src/ng-annotate-main.js

```diff
@@ -1004,11 +1023,27 @@ function judgeInjectArraySuspect(node, ctx) {
 }
 
 function jumpOverIife(node) {
-    let outerfn;
-    if (!(node.type === "CallExpression" && (outerfn = node.callee).type === "FunctionExpression")) {
+    let outerfn = node.callee;
```

Comment:
I think the idea was not to access `node.callee` before the `node.type === "CallExpression"` is verified.

Perhaps just write it like this:

```js
if (node.type !== "CallExpression") {
    return node;
}
const outerfun = node.callee.
if (outerfn.type !== "FunctionExpression" && outerfn.type !== "ArrowFunctionExpression") {
    return node;
}
```